### ISSUE TITLE
Unneeded setNotModified

### ISSUE THREAD

**Issue by hskrasek** [aranw/etags-middleware#3]
I know this package hasn't been updated in quite sometime, but I just noticed today that for the following [lines](https://github.com/aranw/etags-middleware/blob/master/src/Aranw/ETagsMiddleware/ETags.php#L46-L47) you have `if ($response->isNotModified($request))` followed by `$response->setNotModified`. Well in the call for `isNotModified()`, if it was in fact not modified, `setNotModified` is called by `isNotModified`.

Basically, don't need to call the `setNotModified` function again. Man that was a mouthful

**Comment by aranw** [comment]
@hskrasek ah yes your right, I never noticed that. Haven't really updated the package as its so simple theres not been any changes or needed updates but I'll look to make this change, or if you like you can submit a pull request for the change.

**Comment by hskrasek** [comment]
Sorry for the delay @aranw, but pull request submitted.

**Comment by aranw** [comment]
@hskrasek It's alright I've merged it in
